include/abb_librws/v1_0/rws.h

```diff
@@ -179,6 +184,12 @@ namespace abb :: rws :: v1_0
      */
     static const std::string EXCSTATE;
 
+    /**
+     * \brief File path type.
+     */
+    static const std::string FILEPATH;
```

Comment:
Please move this and similar constants to where they belong, since they don't need to be publicly visible. This one should go to `cfg.cpp`.

Reply:
So maybe delete them and e. i.:
Instead:
```cpp
std::string content = FILEPATH + "=" + resource.directory + "/" + resource.filename + "&" + ACTION_TYPE + "=replace";
```
I can just write
```cpp
std::string content = "filepath=" + resource.directory + "/" + resource.filename + "&action-type=replace";
```
what do you think ??

Reply:
I think yes.